config/config.go

```diff
@@ -0,0 +1,22 @@
+package config
+
+import (
+	"os"
+
+	"github.com/joho/godotenv"
+)
+
+func LoadEnv() {
+	env := os.Getenv("APP_ENV")
+
+	if "" == env {
+		env = "dev"
```

Comment:
how do you think about having some sorts of const/var extraction instead of hard-coding the string? i.e:
```
const (
	DevMode   = "dev"
	ReleaseMode = "release"
	TestMode    = "test"
)

...
if "" == env { 
    env = DevMode
}

Reply:
Good catch. Fixed in 4acacf10b55cec9f6ddda7217482556f44f0bf98

Reply:
Fixed syntax error in 9060c968db1aea418cca47ce576424ce423d1bd3